plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -11,7 +11,12 @@
 from functools import partial
 
 from dbt.version import get_installed_version
-from dbt.config.profile import PROFILES_DIR
+
+try:
```

Comment:
Nitpick: Should we consider checking the dbt version rather than blindly trying to import it? It seems more readable and maintainable.

Reply:
Was thinking this too.

dbt has a version tuple we can get from the package (see below):
https://github.com/sqlfluff/sqlfluff/blob/86b553d49bab5fa4a0bb6fd775802b308b7a9bdd/plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py#L40-L42

You can refer here for an example of using a version tuple to handle imports:
https://github.com/sqlfluff/sqlfluff/blob/86b553d49bab5fa4a0bb6fd775802b308b7a9bdd/src/sqlfluff/__init__.py#L8-L12